javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -19,8 +19,11 @@
 package com.github.javaparser.ast.body;
 
 import com.github.javaparser.ast.AccessSpecifier;
+import com.github.javaparser.ast.Modifiers;
+import com.github.javaparser.ast.NodeWithModifiers;
 
 import java.lang.reflect.Modifier;
```

Comment:
and in the current nodes is not treated as a modifier, but as a separate field of the method (see setDefault/isDefault).

It seems therefore that this solution works. Do we agree or do we still see problems with it?